blocks/library/image/index.js

```diff
@@ -82,40 +85,50 @@ registerBlockType( 'core/image', {
 
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, alt, caption } = attributes;
+		const updateAlt = ( newAlt ) => setAttributes( { alt: newAlt } );
+
+		const inspector = focus && (
+			<InspectorControls key="inspector">
+				<TextControl label={ __( 'Alternate Text' ) } value={ alt } onChange={ updateAlt } />
+			</InspectorControls>
+		);
 
 		if ( ! url ) {
 			const uploadButtonProps = { isLarge: true };
 			const setMediaURL = ( media ) => setAttributes( { url: media.url } );
-			return (
+			return [
+				inspector,
```

Comment:
Should we be showing inspector options (Alt Text) when the image is a placeholder?